docs/consuming_recipes.md

```diff
@@ -0,0 +1,127 @@
+# Consuming recipes
+
+Recipes in this repository are evolving continously, contributors are creating pull-requests
+fixing issues and adding new features every day. It is expected that from time to time these
+new recipe revisions stop to work in your project.
```

Comment:
This sounds like CCI is an unstable repository or something like that. Maybe a little bit too alarming. I'd rephrase it with something like "The recipes in this repository are frequently updated to use some features from the latests versions of Conan, and this can make your project to stop working with Conan Center out of the box. Here are a few tips you can follow if that happens".